### PR TITLE
Remove unused code in CZMQToolkit. Fix unit test intermittent failure.

### PR DESCRIPTION
**Pull request by john-gress**

### CODE REVIEW

File: test/CZMQToolkitTests.h [hunk 1]
```diff
@@ -33,64 +30,48 @@ class CZMQToolkitTests : public /*::testing::Test*/ ZeroMQTests {
...
+      ASSERT_NE(nullptr, mRepSocket);
+      zsocket_set_hwm(mRepSocket, 1);
+      zsocket_set_linger(mRepSocket, 0);
+      int bindRetries = 100;
+      while ((zsocket_bind(mRepSocket, mTarget.c_str()) < 0) && bindRetries-- > 0) {
+         zclock_sleep(100);
+      }
+      ASSERT_LT(0, bindRetries);
 
+      mReqSocket = zsocket_new(mContext, ZMQ_REQ);
```
Comment (yannick-polius) [inline test/CZMQToolkitTests.h]: this reads like we are creating a reply socket, then binding it to the target, and using the same socket pointer to do the same binding for a request socket.

unit tests later use this mReqSocket, does it matter to them whether it is pointing to the request or reply socket?

**Comment by yannick-polius** [comment]
:+1:, after question is clarified

**Comment by craig-cogdill** [comment]
:+1:

File: test/CZMQToolkitTests.h (hunk 1 above)
Comment (john-gress) [inline test/CZMQToolkitTests.h]: Clarified the names used here: mReplySocket and mRequestSocket,

**Comment by shwetabhandare** [comment]
:+1:

**Comment by KjellKod** [comment]
+1

**Pull request merged**